scheds/rust/scx_layered/src/main.rs

```diff
@@ -1584,6 +1584,9 @@ impl<'a> Scheduler<'a> {
         skel.struct_ops.layered_mut().exit_dump_len = opts.exit_dump_len;
 
         skel.maps.rodata_data.debug = opts.verbose as u32;
+        // Running scx_layered inside a PID namespace would break the
+        // following.
+        skel.maps.rodata_data.layered_tgid = unsafe { libc::getpid() };
```

Comment:
nit: Pretty sure this is the same and is safe. It may need a cast though and in that case up to you.
```suggestion
        skel.maps.rodata_data.layered_tgid = std::process::id();
```